cpp/ql/src/Security/CWE/CWE-611/XXE.ql

```diff
@@ -35,7 +35,7 @@ module XxeConfig implements DataFlow::StateConfigSig {
   ) {
     // create additional flow steps for `XxeFlowStateTransformer`s
     state2 = node2.asIndirectExpr().(XxeFlowStateTransformer).transform(state1) and
-    DataFlow::simpleLocalFlowStep(node1, node2)
+    DataFlow::simpleLocalFlowStep(node1, node2, _)
```

Comment:
I wonder why this is using `simpleLocalFlowStep` instead of `localFlowStep` 🤔 Oh well, that's for a future PR!